static/app/views/performance/table.tsx

```diff
@@ -422,7 +422,7 @@ class _Table extends Component<Props, State> {
     const prependColumnWidths = ['max-content'];
 
     return (
-      <div>
+      <div data-test-id="PerformanceTable">
```

Comment:
Why CamelCase here, but kabob-case in the other test IDs?

Reply:
lol, will fix it.